images/k8s-cloud-builder/variants.yaml

```diff
@@ -1,10 +1,13 @@
 variants:
+  v1.26-cross1.19-bullseye:
+    CONFIG: 'cross1.19'
+    KUBE_CROSS_VERSION: 'v1.26.0-go1.19.4-bullseye.1'
```

Comment:
this should be `v1.26.0-go1.19.4-bullseye.0` and not with .1 but that is ok

Reply:
I realized that at some point, but I got confused for some reason while updating this image. :sweat_smile: